src/services/user.js

```diff
@@ -9,13 +8,17 @@ import session from './session';
 import storage from './storage';
 
 /**
- * Get all activities for a session, sorted alphabetically by description
+ * Get events attended by the user
  * @param {String} sessionCode Identifier for a session
- * @return {Promise.<User[]>}
+ * @return {Promise.<attendedEvents[]>}
  */
 const getAttendedEvents = (username, termCode) =>
   http.get(`events/chapel/${username}/${termCode}`);
 
+  /**
```

Comment:
This line's indentation is off. The starting `/**` of a documentation block should be indented to match the thing it is documenting. In this case, `const getChapelCredits = ...` is indented zero spaces, so this line should be indented zero spaces as well.